src/main/kotlin/no/nav/tilleggsstonader/sak/opplysninger/oppgave/OppgaveUtil.kt

```diff
@@ -43,6 +43,16 @@ object OppgaveUtil {
         else -> error("Håndterer ikke behandlesAvApplikasjon for $oppgavetype")
     }
 
+    fun skalPlasseresIKlarMappe(oppgavetype: Oppgavetype) = when (oppgavetype) {
+        Oppgavetype.Journalføring,
+        Oppgavetype.BehandleSak,
+        Oppgavetype.BehandleUnderkjentVedtak,
+        Oppgavetype.GodkjenneVedtak,
+        -> true
+
+        else -> error("Håndterer ikke klar-mappe-håndtering for $oppgavetype")
```

Comment:
finnes det ingen som evt. kunne hatt false her?

Reply:
Og hvis vi ikke skal ha en false, er denne sjekken nødvendig?

Reply:
I tillfelle vi oppretter oppgave for en ny type så er det fins hvis vi tar stilling til hvordan den skal håndteres. Eks vurder konsekvens som sannsynligvis skal ha false